common/vcomputer-system.cpp

```diff
@@ -243,7 +266,7 @@ namespace tec {
 		}
 	}
 
-	extern eid active_entity;
+	eid active_entity{ 0 };
```

Comment:
Is this global mutable state?

Reply:
Within vcomputer to indicate which computer for example has focus. Now looking at it it appears we could move this to a class member, good catch.